misc.Rmd

```diff
@@ -65,69 +80,45 @@ citation()
 Calling it with a package name tells you how to cite that package:
 
 ```{r}
-citation("lubridate")
+citation("tidyverse")
```

Comment:
I switched to tidyverse and then noticed that it uses a different -- more modern and minimalist? - approach. We want to move that direction, yeah?

https://github.com/r-lib/usethis/issues/1695